target/linux/ramips/dts/mt7628an_wiznet_wizfi630s.dts

```diff
@@ -81,27 +83,7 @@
 
 &state_default {
 	gpio {
-		groups = "gpio";
-		function = "gpio";
-	};
-
-	i2s {
-		groups = "i2s";
-		function = "gpio";
-	};
-
-	wdt {
-		groups = "wdt";
-		function = "gpio";
-	};
-
-	i2c {
-		groups = "i2c";
-		function = "gpio";
-	};
-
-	refclk {
-		groups = "refclk";
+		groups = "gpio", "wdt", "p1led_an", "p2led_an", "i2s", "i2c", "spis", "pwm0", "pwm1", "refclk", "perst";
```

Comment:
I don't think you will need all of those ...

Reply:
You enabled i2c but disabled i2c pins. Does the i2c work with this setting?

Reply:
> You enabled i2c but disabled i2c pins. Does the i2c work with this setting?
Most likely it will not - i wanted to keep  compatibility with the buttons and leds from the old evaluation board (the module works also fine in it) which has shared pins with i2c and i2s - i think i'll have to move the data to a dtsi file and make separate dts files for both eval boards.

Reply:
i2c_sd is a shared pin with the run (system led) on the eval boards. So it's a either or. Should i2c then be enabled or not?

> You enabled i2c but disabled i2c pins. Does the i2c work with this setting?

Reply:
Based on leds and keys I only get the following here:
i2c p1led_an i2s gpio p2led_an

I don't think we should anything else without a specific reason.

Reply:
pwm0, pwm1 and uart2 pins are also available. These are not labelled explicitly, but available on pin headers on the evalboard. Should these then be enabled?

Reply:
If the feature can be used (Have you tested pwm and uart BTW?), enable the feature node. If the pin is used for other purpose (like led), then disable the feature node and set the pin as gpio.

Reply:
In one point I was wrong, the pins are just on the module, but not on the evalation board. So I cannot test without a customized mother board.